hoover/site/urls.py

```diff
@@ -22,10 +22,23 @@
         url(r'^invitation/(?P<code>.*)$', twofactor_views.invitation),
         url(r'^accounts/login/$', auth_views.login, kwargs={
             'authentication_form': twofactor_views.AuthenticationForm}),
+        url(r'^accounts/', include('django.contrib.auth.urls')),
+    ]
+
+elif installed.oauth2:
```

Comment:
Could we somehow fool-proof the case when the admin has configured both oauth2 AND twofactor auth?

Maybe throw an exception if both are installed, so it the server even start?

Also, we should probably document somewhere that these are mutually exclusive, even if it's obvious after you think about it.